app/uk/gov/hmrc/tai/service/TaxCodeChangeService.scala

```diff
@@ -142,7 +142,7 @@ class TaxCodeChangeServiceImpl @Inject()(
     }
 
   def latestTaxCodes(nino: Nino, taxYear: TaxYear)(implicit hc: HeaderCarrier): Future[Seq[TaxCodeSummary]] =
-    taxCodeChangeConnector.taxCodeHistory(nino, taxYear, taxYear).map { taxCodeHistory =>
+    taxCodeChangeConnector.taxCodeHistory(nino, taxYear, taxYear.next).map { taxCodeHistory =>
```

Comment:
Should it be the call to the private method?

Reply:
We tried this before and it didnt work for some reason however I have changed it there and it works fine